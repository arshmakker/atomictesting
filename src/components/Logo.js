import React from 'react'

function Logo({
    classlogo="",
    srcImg = "https://img.icons8.com/ios/50/000000/pick-up.png"

}) {    
    return (
        <img className={classlogo} src={srcImg} alt="unapologetic logo"/>
    )
}

export default Logo;