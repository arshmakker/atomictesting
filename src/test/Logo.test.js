import React from 'react'
import Logo from '../components/Logo'
import renderer from 'react-test-renderer'

const classlogo = "sampleClass"
const srcImg = "https://img.icons8.com/search"


it("Renders the Logo correctly with no props ", () => {
    const tree = renderer
        .create(<Logo/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
})
it("Renders the Logo correctly with test props", () => {

    const tree = renderer
        .create(<Logo classlogo={classlogo}
            srcImg={srcImg} ></Logo>)
        .toJSON();
    expect(tree).toMatchSnapshot();
})