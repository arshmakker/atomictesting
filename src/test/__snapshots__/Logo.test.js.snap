// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the Logo correctly with no props  1`] = `
<img
  alt="unapologetic logo"
  className=""
  src="https://img.icons8.com/ios/50/000000/pick-up.png"
/>
`;

exports[`Renders the Logo correctly with test props 1`] = `
<img
  alt="unapologetic logo"
  className="sampleClass"
  src="https://img.icons8.com/search"
/>
`;
